src/security.py

```diff
@@ -154,7 +154,7 @@ def _limit_view_function(view_name, limit_func):
     app.view_functions[view_name] = limit_func(app.view_functions[view_name])
 _limit_view_function('security.login', limiter.limit('10/hour'))
 if app.config['SECURITY_REGISTERABLE']:
-    _limit_view_function('security.register', limiter.limit('10/hour'))
```

Comment:
this is not required as *only* admins will use the web portal

Reply:
so should i delete the "limiter.limit('10/hour')" part of the security.login?

Reply:
you should not change the limits on security.register/login

Reply:
changed it back security.register to 10/hr